docs/source/reporters.rst

```diff
@@ -231,33 +232,37 @@ You can have notifications sent to you through the `Matrix protocol`_.
 To achieve this, you first need to register a Matrix account for the bot
 on any homeserver.
 
-You then need to acquire an access token and room ID, using the
-following instructions adapted from `this
-guide <https://t2bot.io/docs/access_tokens/>`__:
-
-1. Open `Riot.im <https://riot.im/app/>`__ in a private browsing window
-2. Register/Log in as your bot, using its user ID and password.
-3. Set the display name and avatar, if desired.
-4. In the settings page, select the "Help & About" tab, scroll down to the bottom and click Access
-   Token: <click to reveal>.
-5. Copy the highlighted text to your configuration.
-6. Join the room that you wish to send notifications to.
-7. Go to the Room Settings (gear icon) and copy the *Internal Room ID*
-   from the bottom.
-8. Close the private browsing window **but do not log out, as this
-   invalidates the Access Token**.
+You then need to acquire the room ID, using the following instructions:
+
+#. Open `Riot.im <https://riot.im/app/>`__ in a private browsing window
```

Comment:
Is there some command-line matrix client that can be used for this without relying on `riot.im` or a similar service? I know that this was in here before, but since one of the advantages of this change is that rooms are now encrypted, the encryption would be kinda pointless if initially we rely on trusting `riot.im`.

Reply:
> Is there some command-line matrix client that can be used for this without relying on riot.im or a similar service?

According to spec, room `name` is not unique, room `alias` is optional so `room ID` in general is the only reliable room identifier (https://matrix.org/docs/spec/client_server/latest#m-room-name). `room ID` is obtained during room creation and stored by the client. Any client used to create room should be able to display `room ID` (I can note in docs that ~riot.im~ element is just an example)

>the encryption would be kinda pointless if initially we rely on trusting riot.im

Why?